app/connectors/connectors/sql/sql-common/pom.xml

```diff
@@ -76,6 +76,11 @@
       <groupId>org.apache.derby</groupId>
       <artifactId>derby</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.postgresql</groupId>
+      <artifactId>postgresql</artifactId>
+      <version>${postgresql.version}</version>
```

Comment:
Can we add <scope>test</scope> please or it this already setup in some dependency management ?

Reply:
Same for derby btw